LoopUI/Views/SettingsView.swift

```diff
@@ -0,0 +1,207 @@
+                    if viewModel.showWarning {
+                        Spacer()
+                        Text(LocalizedString("⚠️", comment: "Warning symbol"))
+                    }
+                }
+            }
+        }
+    }
+        
+    private var therapySettingsSection: some View {
+        Section (header: SectionHeader(label: LocalizedString("Configuration", comment: "The title of the Configuration section in settings"))) {
+            NavigationLink(destination: Text("Therapy Settings")) {
+                LargeButton(action: { },
+                            includeArrow: false,
+                            image: Image(bundleString: "Therapy Icon"),
+                            label: LocalizedString("Therapy Settings", comment: "Title text for button to Therapy Settings"),
+                            details: LocalizedString("Diabetes Treatment", comment: "Descriptive text for Therapy Settings"))
+            }
+        }
+    }
+    
+    private var deviceSettingsSection: some View {
+        Section {
+            pumpSection
+            cgmSection
+        }
+    }
+    
+    private var pumpSection: some View {
+        if viewModel.pumpManagerSettingsViewModel.isSetUp {
+            // TODO: this "dismiss then call onTapped()" here is temporary, until we've completely gotten rid of SettingsTableViewController
+            return LargeButton(action: { self.dismiss(); self.viewModel.pumpManagerSettingsViewModel.onTapped() },
+                               image: Image(uiImage: viewModel.pumpManagerSettingsViewModel.image, bundleString: "Omnipod"),
+                               label: viewModel.pumpManagerSettingsViewModel.name,
+                               details: viewModel.pumpManagerSettingsViewModel.details)
+        } else {
+            // TODO: this "dismiss then call onTapped()" here is temporary, until we've completely gotten rid of SettingsTableViewController
+            return LargeButton(action: { self.dismiss(); self.viewModel.pumpManagerSettingsViewModel.onTapped() },
+                               image: Image(bundleString: "Omnipod"),
+                               label: LocalizedString("Add Pump", comment: "Title text for button to add pump device"),
+                               details: LocalizedString("Tap here to set up a pump", comment: "Descriptive text for button to add pump device"))
+        }
+    }
+    
+    private var cgmSection: some View {
+        if viewModel.cgmManagerSettingsViewModel.isSetUp {
+            // TODO: this "dismiss then call onTapped()" here is temporary, until we've completely gotten rid of SettingsTableViewController
+            return LargeButton(action: { self.dismiss(); self.viewModel.cgmManagerSettingsViewModel.onTapped() },
+                               image: Image(uiImage: viewModel.cgmManagerSettingsViewModel.image, bundleString: "Dexcom G6"),
```

Comment:
And same question re "Dexcom G6" here and below

Reply:
see above :)